bears/js/HappinessLintBear.py

```diff
@@ -20,5 +20,14 @@ class HappinessLintBear:
     CAN_DETECT = {'Syntax'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         use_spaces: bool=False):
+
+        if use_spaces:
+            raise ValueError(
+                'Set "use_spaces=False" for HappinessLintBear '
+                'as "use_spaces=Ture" is incompatible '
+                'for HappinessLintBear.'
```

Comment:
1. incompatible ~~as~~ **with**
2. `Ture` --> `True`
3. Try to get the bear name from `self.name` and use a format string.
3. A bit too much text, rather
   ```
   "use_spaces=True" is incompatible with {}, please set it to false.
   ```
   or similar

Reply:
i dont think self is defined in this case.

Reply:
Oh right, but you'll find out where to get the name from ;)

Reply:
but i dont think theres a need to do so as the file itself is HappinessLnitBear why to find name from somewhere

Reply:
if someone changes the name, it's automatically adapted. It's important to use the same "source" of data, so that a change in one place properly reflects everywhere without the need to do manual modifications everywhere (and forgetting things).

Reply:
okk but i cannot find the place where we can retrieve the name from! ;P

Reply:
Can we get some help from the decorator?

Reply:
afaik it also cant help in accessing the name. :)

Reply:
-->
```
@classmethod
def create_arguments(cls, ...):
    ...
    cls.name
```

Reply:
done ;P

Reply:
btw how does it matter? @Makman2  :D

Reply:
always use a single source of the same information, maintainability matters ;)